README.md

```diff
@@ -1,70 +1,55 @@
-# Getting Started with Create React App
+# Math magicians
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Frist Microverse projects to get start with React.
+The math magician web app is a simple calculator for solving minor math problems.
 
-## Available Scripts
+## Screenshoot
 
-In the project directory, you can run:
+![Screeshot](./src/Screenshot.png)
```

Comment:
- [OPTIONAL ] The screenshot picture does not display correctly, Kindly fix this.